feature-libs/cart/import-export/core/services/import.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { Observable, Observer } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ImportService {
```

Comment:
Don't you think ImportService is quite too general name? And import-export not too specific lib for such a generic util service?

Reply:
I was considering `import text file`, but the current state allows to fine extend this service in the future if necessary with new methods. By us or by customer